apps/src/templates/GradientNavCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ProgressButton from './progress/ProgressButton';
+
+const styles = {
+  card: {
+    flex: 1,
```

Comment:
http://caniuse.com/#feat=flexbox

Worth verifying this works as expected in IE11.